src/inc/palclr.h

```diff
@@ -484,7 +484,14 @@
 //  and it provides __declspec(selectany) to instruct the linker to merge
 //  duplicate external const static data copies into one.
 //  
+#if defined(__clang__)
+#define SELECTANY extern
```

Comment:
It is strange that we don't get this error while compiling on Linux from other places where we have the same definition of SELECTANY and use it for const arrays. I would like to eventually investigate it, but not for this change.
Anyways, the palclr.h is not used on Unix at all - its contents is all wrapped in  `#ifndef FEATURE_PAL`, which is defined for Unix only, since only Unix uses PAL. So using the __clang__ define here is actually ok.